### PR TITLE
DUOS-2172[risk=no] Remove old DAR Voting and Review pages

### PR DESCRIPTION
**Pull request by JVThomas**
Addresses [DUOS-2172](https://broadworkbench.atlassian.net/browse/DUOS-2172)

PR removes the old DAR voting and review pages alongside any associated ajax and helper methods (that are not used elsewhere).

Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/libs/ajax.js [hunk 1]
```diff
@@ -419,23 +413,7 @@ export const Election = {
...
-
-export const Email = {
-
-  sendReminderEmail: async (voteId) => {
-    const url = `${await getApiUrl()}/api/emailNotifier/reminderMessage/${voteId}`;
-    const res = await fetchOk(url, fp.mergeAll([Config.authOpts(), { method: 'POST' }]));
-    return res;
   }
-
 };
```
Comment (rushtong) [inline src/libs/ajax.js]: We have a ticket in the sprint to use this function ... might not want to delete it quite yet.

**Review by rushtong — APPROVED** [review]
One minor suggestion, then 👍🏽

**Review by quazi-h — APPROVED** [review]
Code removal looks good. I can no longer access the access_review or review_results pages for collections when running these changes locally.
